Makefile

```
@@ -55,4 +52,4 @@ testall:
 	@go test ./...
 
 bench:
-	@go test ./pkg/runners/zero -bench=. -cpuprofile $(CPU_PROFILE) -memprofile $(MEM_PROFILE)
+	@go run scripts/benchmark.go --pkg=${PKG_NAME} --test=${TEST}
```

```suggestion
@echo "Running benchmarks.."
@go run scripts/benchmark.go --pkg=${PKG_NAME} --test=${TEST}
```
would make it consistent with other make commends